src/styles/card.scss

```diff
@@ -0,0 +1,71 @@
+.card {
+  width: 200px;
+  border: 1px solid white;
```

Comment:
According to the [Code Style Rules](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), the border color should be consistent with the design. If `white` is not the color used in the design, consider using the appropriate color variable or hex code.